.github/workflows/pull-request.yaml

```diff
@@ -54,11 +54,10 @@ jobs:
   lint-unit:
     name: Lint checkers and unit tests
     uses: ./.github/workflows/_lint-unit.yaml
-    # Note(rgildein): Lint checkers and unit tests should be triggered on any push, any
-    # pull requested, approval pull request review was submitted or review comment with
+    # Note(rgildein): Lint checkers and unit tests should be triggered on any push (charm-release),
```

Comment:
I think that we can remove any mention about push and triggered this workflow only in pull_request and pull_request_review. Can I ask you to remove this [line](https://git.launchpad.net/bootstack-charms-spec/tree/templates/charm/.github/workflows/pr.yaml.j2#n5) from our templates.